haproxy/haproxy.cfg

```diff
@@ -64,6 +64,3 @@ backend couchdb-servers
   retry-on all-retryable-errors
   log global
   retries 5
-  server couchdb1 $COUCHDB1_SERVER:5984 check inter 2s
```

Comment:
Add a comment or something here to explain how the couchdb servers get appended.